.gitignore

```diff
@@ -387,4 +387,9 @@ xcuserdata
 .idea/encodings.xml
 .idea/misc.xml
 
-scripts/pub_publish.sh
\ No newline at end of file
+scripts/pub_publish.sh
+.idea/
+.flutter-plugins-dependencies
+.gradle/
+local.properties
+.flutter-plugins
```

Comment:
```
.idea/encodings.xml
.idea/misc.xml	.idea/misc.xml
```
can be removed and personally I'd like to add a new line at the EoF .

Reply:
Updated!